src/coderunner/useCodeRunner.tsx

```diff
@@ -0,0 +1,202 @@
+import React, {
```

Comment:
React component wrapping the new code runner object.
Objects using useCodeRunner share a singleton runner instance at the moment, and can provide handlers to common callbacks (e.g. draw with turtle, play sound, print etc.)